okhttp-logging-interceptor/src/main/kotlin/okhttp3/logging/HttpLoggingInterceptor.kt

```diff
@@ -211,7 +212,7 @@ class HttpLoggingInterceptor @JvmOverloads constructor(
           }
         }
 
-        val charset: Charset = requestBody.contentType().charset()
+        val charset: Charset = requestBody.contentType().charsetNonNull()
```

Comment:
Isn’t !! the way to go here?

Reply:
I'm not clear, we know it's non null here.  Will follow up if you are convinced.